File: taobao-shopcart-interface/src/main/java/com/taobao/shopcart/service/CartItemService.java
package com.taobao.shopcart.service;

import com.taobao.pojo.group.Cart;
import com.taobao.pojo.group.CartItem;

import java.util.List;

public interface CartItemService {

    /**
     * 添加用户的购物列表到redis
     */
    public void add(List<CartItem> cartItemList,Long userId);

    /**
     * 从redis中取出用户最近添加的购物项列表
     */
    public List<CartItem>  getCartItemList(Long userId);

    /**
     *
     */
}

File: taobao-user-web/src/main/webapp/js/service/cartItemService.js
app.service('cartItemService',function ($http) {

    //将用户选择的购物项添加到redis用于在getOrderInfo.html界面显示
    this.addCartItem = function (cartItemList) {
        alert(cartItemList);
        return $http.post("cartItem/add.do",cartItemList);
    }
    /*location.href="http://localhost:8081/pay.html#?orderId = "+orderId;*/
    //将redis中取出用户的购物项列表用于在getOrderInfo.html界面显示
    this.getCartItemList = function (userId) {
        return $http.get("cartItem/get.do?userId="+userId);
    }
});